config/spamd/spamd_db.php

```diff
@@ -122,35 +119,38 @@
 }
 
 /* spam trap e-mail address */
-if($_GET['spamtrapemail'] <> "") {
```

Comment:
Well, I just wanted to ask since perhaps it might have been added on purpose. Doesn't seem so. Thanks! ;)